### ISSUE TITLE
WooCommerce Extensions fail to update leading to confusing UX loop

### ISSUE THREAD

**Issue by robertdall** [woocommerce/woocommerce#19831]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [ ] The issue still exists against the latest `master` branch of WooCommerce on Github (this is **not** the same version as on WordPress.org!)
- [x] I have attempted to find the simplest possible steps to reproduce the issue
- [ ] I have included a failing test as a pull request (Optional)

## Steps to reproduce the issue

In WooCommerce 3.3.5 and WordPress 4.9.5 I was trying to update extensions. 

I would try to click update on an extension and it would redirect me to plugin update page and then it would immediately tell me the plugin was updates to the latest version immediately. It also directed me back to the Plugins page and not WooCommerce extensions page. If I tried to update the plugin again it would send me on the same loop.

It was only after I clicked the update button on the extensions page that it said: _Authentication and Subscriptions caches refreshed successfully_ then I was actually able to update the plugin. 

See Screencast of this experience:
https://cloudup.com/cfCT7qw5ELr

While I was able to update all three plugins in this work around method, this is far from a great user experience for the following reasons. 

1. There is no connection, no notification  in the tried and true Plugin page for WordPress Plugins. The only way you know there is an update to these plugins is the red number notification on the drop down for extensions. Hypothetically if I didn't hover over the WooCommerce tab I would never see the notification for plugin updates on these extensions. 

2. You are redirected to the Plugin page even after you update the WooComm extension. Which has no listing and no reference to any WooComm extensions. I realize there are reasons why developers put the plugins in the Extensions page as appose to plugins but to the average user of WooCommerce this would be very confusing. Can we either have one or the other? Not a poorly handled combination of both?

3. When I tried contacting support to report this, I couldn't even report this as a  support ticket, I was immediately directed to live chat where I was to go to WooCommerce.com and search for if the plugin is "outdated". There was no mention of outdated anywhere in my entire experience, live chat agent didn't give up and told me it's because my plugins were "outdated" what ever that means. (I never saw an reference to outdated anywhere in my WooCommerce.com experience and yes I clicked that down arrow and did a find word in browser)

What this experience shows me and why I am reporting it here is how misplaced or lack of UX can lead to odd and often frustrating support experiences that don't even have to happen with the proper user experience.

Having to debate with support what I was seeing was just a poor support tech. While I am a huge fan of live chat, this could have been handled better via long form support ticket. So I am reporting it here where I don't have to debate with support and might get the eyes of someone who will actually listen.

Again this is not a support request but an example of poor user experience from start to finish.

- [x] This bug happens with only WooCommerce plugin active
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/)
- [ ] I can reproduce this bug consistently using the steps above

## WordPress Environment

<details>
`
### WordPress Environment ###

Home URL: https://store.artwolfe.com
Site URL: https://store.artwolfe.com
WC Version: 3.3.5
Log Directory Writable: ✔
WP Version: 4.9.5
WP Multisite: ✔
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: Apache
PHP Version: 5.6.31-4+wpengine10
PHP Post Max Size: 100 MB
PHP Time Limit: 3600
PHP Max Input Vars: 1000
cURL Version: 7.35.0
OpenSSL/1.0.1f

SUHOSIN Installed: –
MySQL Version: 5.6.39
Max Upload Size: 50 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.3.5
WC Database Prefix: wp_2_
Total Database Size: 280.75MB
Database Data Size: 170.89MB
Database Index Size: 109.86MB
wp_2_woocommerce_sessions: Data: 10.05MB + Index: 0.13MB
wp_2_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_2_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_2_woocommerce_downloadable_product_permissions: Data: 0.14MB + Index: 0.23MB
wp_2_woocommerce_order_items: Data: 1.52MB + Index: 0.17MB
wp_2_woocommerce_order_itemmeta: Data: 3.52MB + Index: 4.03MB
wp_2_woocommerce_tax_rates: Data: 0.05MB + Index: 0.06MB
wp_2_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.05MB
wp_2_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_2_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.05MB
wp_2_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_2_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_2_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_2_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_2_apto_sort_list: Data: 0.02MB + Index: 0.02MB
wp_2_commentmeta: Data: 0.30MB + Index: 0.30MB
wp_2_comments: Data: 3.52MB + Index: 2.08MB
wp_2_failed_jobs: Data: 0.02MB + Index: 0.00MB
wp_2_gf_addon_feed: Data: 0.02MB + Index: 0.02MB
wp_2_links: Data: 0.02MB + Index: 0.02MB
wp_2_mailchimp_carts: Data: 0.05MB + Index: 0.00MB
wp_2_options: Data: 6.50MB + Index: 0.50MB
wp_2_postmeta: Data: 25.56MB + Index: 24.06MB
wp_2_posts: Data: 2.52MB + Index: 1.11MB
wp_2_queue: Data: 0.02MB + Index: 0.00MB
wp_2_rg_form: Data: 0.02MB + Index: 0.00MB
wp_2_rg_form_meta: Data: 0.14MB + Index: 0.00MB
wp_2_rg_form_view: Data: 1.52MB + Index: 0.67MB
wp_2_rg_incomplete_submissions: Data: 0.02MB + Index: 0.02MB
wp_2_rg_lead: Data: 0.42MB + Index: 0.11MB
wp_2_rg_lead_detail: Data: 1.52MB + Index: 2.20MB
wp_2_rg_lead_detail_long: Data: 0.02MB + Index: 0.00MB
wp_2_rg_lead_meta: Data: 0.25MB + Index: 0.38MB
wp_2_rg_lead_notes: Data: 0.02MB + Index: 0.03MB
wp_2_rg_mailchimp: Data: 0.02MB + Index: 0.02MB
wp_2_termmeta: Data: 0.02MB + Index: 0.03MB
wp_2_terms: Data: 0.02MB + Index: 0.03MB
wp_2_term_relationships: Data: 0.14MB + Index: 0.11MB
wp_2_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_2_tnwoo_log2: Data: 19.52MB + Index: 0.00MB
wp_2_wc_download_log: Data: 0.02MB + Index: 0.03MB
wp_2_wc_webhooks: Data: 0.02MB + Index: 0.02MB
wp_2_yoast_seo_links: Data: 0.02MB + Index: 0.02MB
wp_2_yoast_seo_meta: Data: 0.02MB + Index: 0.00MB
wp_4_apto_sort_list: Data: 0.02MB + Index: 0.02MB
wp_4_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_4_comments: Data: 0.02MB + Index: 0.08MB
wp_4_gf_addon_feed: Data: 0.02MB + Index: 0.02MB
wp_4_links: Data: 0.02MB + Index: 0.02MB
wp_4_options: Data: 4.45MB + Index: 0.17MB
wp_4_postmeta: Data: 3.52MB + Index: 0.89MB
wp_4_posts: Data: 1.52MB + Index: 0.34MB
wp_4_rg_form: Data: 0.02MB + Index: 0.00MB
wp_4_rg_form_meta: Data: 0.11MB + Index: 0.00MB
wp_4_rg_form_view: Data: 1.52MB + Index: 0.61MB
wp_4_rg_incomplete_submissions: Data: 0.02MB + Index: 0.02MB
wp_4_rg_lead: Data: 0.48MB + Index: 0.13MB
wp_4_rg_lead_detail: Data: 1.52MB + Index: 2.36MB
wp_4_rg_lead_detail_long: Data: 0.02MB + Index: 0.00MB
wp_4_rg_lead_meta: Data: 0.31MB + Index: 0.42MB
wp_4_rg_lead_notes: Data: 0.02MB + Index: 0.03MB
wp_4_termmeta: Data: 0.02MB + Index: 0.03MB
wp_4_terms: Data: 0.02MB + Index: 0.03MB
wp_4_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_4_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_4_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_4_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.03MB
wp_4_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB
wp_4_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB
wp_4_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_4_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.05MB
wp_4_woocommerce_termmeta: Data: 0.02MB + Index: 0.03MB
wp_4_yoast_seo_links: Data: 0.19MB + Index: 0.06MB
wp_4_yoast_seo_meta: Data: 0.06MB + Index: 0.00MB
wp_5_apto_sort_list: Data: 0.02MB + Index: 0.02MB
wp_5_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_5_comments: Data: 0.02MB + Index: 0.08MB
wp_5_gf_addon_feed: Data: 0.02MB + Index: 0.02MB
wp_5_links: Data: 0.02MB + Index: 0.02MB
wp_5_options: Data: 2.47MB + Index: 0.14MB
wp_5_postmeta: Data: 0.45MB + Index: 0.16MB
wp_5_posts: Data: 0.25MB + Index: 0.06MB
wp_5_rg_form: Data: 0.02MB + Index: 0.00MB
wp_5_rg_form_meta: Data: 0.02MB + Index: 0.00MB
wp_5_rg_form_view: Data: 1.52MB + Index: 0.36MB
wp_5_rg_incomplete_submissions: Data: 0.02MB + Index: 0.02MB
wp_5_rg_lead: Data: 0.14MB + Index: 0.03MB
wp_5_rg_lead_detail: Data: 0.17MB + Index: 0.38MB
wp_5_rg_lead_detail_long: Data: 0.02MB + Index: 0.00MB
wp_5_rg_lead_meta: Data: 0.13MB + Index: 0.19MB
wp_5_rg_lead_notes: Data: 0.02MB + Index: 0.03MB
wp_5_rg_mailchimp: Data: 0.02MB + Index: 0.02MB
wp_5_termmeta: Data: 0.02MB + Index: 0.03MB
wp_5_terms: Data: 0.02MB + Index: 0.03MB
wp_5_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_5_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_5_yoast_seo_links: Data: 0.02MB + Index: 0.02MB
wp_5_yoast_seo_meta: Data: 0.02MB + Index: 0.00MB
wp_6_apto_sort_list: Data: 0.02MB + Index: 0.02MB
wp_6_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_6_comments: Data: 0.02MB + Index: 0.08MB
wp_6_gf_addon_feed: Data: 0.02MB + Index: 0.02MB
wp_6_links: Data: 0.02MB + Index: 0.02MB
wp_6_options: Data: 2.50MB + Index: 0.14MB
wp_6_postmeta: Data: 0.08MB + Index: 0.03MB
wp_6_posts: Data: 0.13MB + Index: 0.06MB
wp_6_rg_form: Data: 0.02MB + Index: 0.00MB
wp_6_rg_form_meta: Data: 0.02MB + Index: 0.00MB
wp_6_rg_form_view: Data: 0.47MB + Index: 0.33MB
wp_6_rg_incomplete_submissions: Data: 0.02MB + Index: 0.02MB
wp_6_rg_lead: Data: 0.06MB + Index: 0.03MB
wp_6_rg_lead_detail: Data: 0.06MB + Index: 0.06MB
wp_6_rg_lead_detail_long: Data: 0.02MB + Index: 0.00MB
wp_6_rg_lead_meta: Data: 0.05MB + Index: 0.05MB
wp_6_rg_lead_notes: Data: 0.02MB + Index: 0.03MB
wp_6_termmeta: Data: 0.02MB + Index: 0.03MB
wp_6_terms: Data: 0.02MB + Index: 0.03MB
wp_6_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_6_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_6_yoast_seo_links: Data: 0.02MB + Index: 0.02MB
wp_6_yoast_seo_meta: Data: 0.02MB + Index: 0.00MB
wp_7_apto_sort_list: Data: 0.02MB + Index: 0.02MB
wp_7_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_7_comments: Data: 0.02MB + Index: 0.08MB
wp_7_gf_addon_feed: Data: 0.02MB + Index: 0.02MB
wp_7_links: Data: 0.02MB + Index: 0.02MB
wp_7_options: Data: 3.48MB + Index: 0.14MB
wp_7_postmeta: Data: 0.27MB + Index: 0.09MB
wp_7_posts: Data: 0.17MB + Index: 0.06MB
wp_7_rg_form: Data: 0.02MB + Index: 0.00MB
wp_7_rg_form_meta: Data: 0.08MB + Index: 0.00MB
wp_7_rg_form_view: Data: 0.36MB + Index: 0.27MB
wp_7_rg_incomplete_submissions: Data: 0.02MB + Index: 0.02MB
wp_7_rg_lead: Data: 0.09MB + Index: 0.03MB
wp_7_rg_lead_detail: Data: 0.11MB + Index: 0.25MB
wp_7_rg_lead_detail_long: Data: 0.02MB + Index: 0.00MB
wp_7_rg_lead_meta: Data: 0.06MB + Index: 0.08MB
wp_7_rg_lead_notes: Data: 0.02MB + Index: 0.03MB
wp_7_termmeta: Data: 0.02MB + Index: 0.03MB
wp_7_terms: Data: 0.02MB + Index: 0.03MB
wp_7_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_7_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_7_yoast_seo_links: Data: 0.02MB + Index: 0.02MB
wp_7_yoast_seo_meta: Data: 0.02MB + Index: 0.00MB
wp_apto_sort_list: Data: 0.02MB + Index: 0.02MB
wp_blogs: Data: 0.02MB + Index: 0.03MB
wp_blog_versions: Data: 0.02MB + Index: 0.02MB
wp_commentmeta: Data: 0.48MB + Index: 0.23MB
wp_comments: Data: 1.52MB + Index: 0.45MB
wp_domain_mapping: Data: 0.02MB + Index: 0.02MB
wp_domain_mapping_logins: Data: 0.02MB + Index: 0.00MB
wp_gf_addon_feed: Data: 0.02MB + Index: 0.02MB
wp_links: Data: 0.02MB + Index: 0.02MB
wp_options: Data: 3.47MB + Index: 0.17MB
wp_postmeta: Data: 5.52MB + Index: 3.03MB
wp_posts: Data: 4.52MB + Index: 0.78MB
wp_quotescollection: Data: 0.02MB + Index: 0.00MB
wp_registration_log: Data: 0.02MB + Index: 0.02MB
wp_rg_form: Data: 0.02MB + Index: 0.00MB
wp_rg_form_meta: Data: 0.17MB + Index: 0.00MB
wp_rg_form_view: Data: 3.52MB + Index: 3.03MB
wp_rg_incomplete_submissions: Data: 0.02MB + Index: 0.02MB
wp_rg_lead: Data: 2.52MB + Index: 0.38MB
wp_rg_lead_detail: Data: 4.52MB + Index: 9.06MB
wp_rg_lead_detail_long: Data: 0.42MB + Index: 0.00MB
wp_rg_lead_meta: Data: 1.52MB + Index: 3.34MB
wp_rg_lead_notes: Data: 0.02MB + Index: 0.03MB
wp_rg_mailchimp: Data: 0.02MB + Index: 0.02MB
wp_signups: Data: 0.02MB + Index: 0.06MB
wp_site: Data: 0.02MB + Index: 0.02MB
wp_sitemeta: Data: 0.38MB + Index: 0.03MB
wp_stream: Data: 0.19MB + Index: 0.38MB
wp_stream_context_tmp: Data: 4.52MB + Index: 7.55MB
wp_stream_meta: Data: 10.02MB + Index: 14.95MB
wp_stream_tmp: Data: 14.55MB + Index: 11.58MB
wp_termmeta: Data: 0.02MB + Index: 0.03MB
wp_terms: Data: 0.13MB + Index: 0.16MB
wp_term_relationships: Data: 0.33MB + Index: 0.30MB
wp_term_taxonomy: Data: 0.11MB + Index: 0.16MB
wp_usermeta: Data: 5.52MB + Index: 7.03MB
wp_users: Data: 0.33MB + Index: 0.30MB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_termmeta: Data: 0.02MB + Index: 0.03MB
wp_yoast_seo_links: Data: 0.09MB + Index: 0.02MB
wp_yoast_seo_meta: Data: 0.02MB + Index: 0.00MB

### Post Type Counts ###

apto_sort: 10
attachment: 843
nav_menu_item: 14
page: 10
post: 86
product: 444
product_variation: 1408
revision: 5
shop_coupon: 137
shop_order: 3428
shop_order_refund: 32
wp-help: 49

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (36) ###

Gravity Forms MailChimp Add-On: by rocketgenius – 4.2
MailChimp for WooCommerce: by MailChimp – 2.1.5 – Not tested with the active version of WooCommerce
Menu Social Icons: by Paul Clark – 1.3.10
Search By SKU - for Woocommerce: by Matthew Lawson – 0.6.1 – Not tested with the active version of WooCommerce
WooCommerce Payment Gateway Based Fees: by WooThemes – 3.1.4
WooCommerce Print Invoice & Delivery Note: by Tyche Softwares – 4.4.2
WooCommerce Digital Download Email: by SkyVerge
Sergey Biryukov – 0.1 – Not tested with the active version of WooCommerce

WooCommerce Authorize.Net AIM Gateway: by SkyVerge – 3.14.1
WooCommerce Google Analytics Integration: by WooCommerce – 1.4.4
WooCommerce Quantity Increment: by Automattic
WooThemes – 1.1.0 – Not tested with the active version of WooCommerce

WooCommerce FedEx Shipping: by WooCommerce – 3.4.10
WooCommerce USPS Shipping: by WooCommerce – 4.4.16
WooCommerce Shortcodes: by WooThemes
Claudio Sanches – 1.0.0 – Not tested with the active version of WooCommerce

WooCommerce Smart Coupons: by WooCommerce – 3.4.0
WooCommerce: by Automattic – 3.3.5
WP Hide Post: by scriptburn.com – 2.0.10
YITH Purchased Column for WooCommerce: by YITHEMES – 1.1.2
Art Wolfe Image Widget: by 32spokes – 1.0.0
Menu Social Icons: by Paul Clark – 1.3.10
Menu Humility: by Mark Jaquith – 0.3.1
IPTC Caption Mapping: by 32spokes – 1.0.0
WP Help: by Mark Jaquith – 1.5.4
Unattach and Re-attach Media Attachments: by davidn.de – 1.2
Login Logo: by Mark Jaquith – 0.9.0
Network Nav Menus: by IvyCat Web Services – 1.0.0
WP Query Multisite: by Eric Lewis
Miguel Peixe
Sergey Biryukov – 1.0

Advanced Image Styles: by Gregory Cornelius – 0.4.1
Simple Local Avatars: by Jake Goldman
10up – 2.0

Advanced Post Types Order: by Nsp Code – 4.0.8
Black Studio TinyMCE Widget: by Black Studio – 2.6.2
Gravity Forms: by rocketgenius – 2.2.6
Yoast SEO: by Team Yoast – 7.3
Jetpack by WordPress.com: by Automattic – 5.3
Manual Control for Jetpack: by Mark Jaquith – 0.2
Stream: by XWP – 3.2.2
SendGrid: by SendGrid – 1.9.4

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

### WC Pages ###

Shop base: #4 - /shop/
Cart: #5 - /cart/
Checkout: #6 - /checkout/
My account: #7 - /my-account/
Terms and conditions: #398 - /terms-conditions/

### Theme ###

Name: Art Wolfe Store
Version: 0.1
Author URL: http://32spokes.com
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: artwolfe-store/woocommerce/cart/cart-empty.php
artwolfe-store/woocommerce/content-product.php
artwolfe-store/woocommerce/content-product_cat.php
artwolfe-store/woocommerce/emails/customer-processing-order.php
artwolfe-store/woocommerce/emails/email-footer.php
artwolfe-store/woocommerce/emails/email-header.php
artwolfe-store/woocommerce/product-searchform.php version 2.5.0 is out of date. The core version is 3.3.0
artwolfe-store/woocommerce/single-product/price.php

Outdated Templates: ❌Learn how to update

### Authorize.Net AIM ###

Environment: Production
Debug Mode: Save to Log
`
</details>

**Comment by mikejolley** [comment]
Seems legit. @garymurray @kovshenin Is this a caching issue or something woo.com side?

**Comment by mikejolley** [comment]
The update error was a bug so I think the UX issue is solved if that is fixed. Updates usually appear on both screens.

**Comment by robertdall** [comment]
@mikejolley Ok will report back if I don't see them in both screens in future updates.

**Comment by mikejolley** [comment]
See https://github.com/woocommerce/woocommerce/pull/19842

**Comment by robertdall** [comment]
@mikejolley I updated my WooComm install to 3.4.1 and I had the same issue. Here is a video which replicates the previous issue.  https://cloudup.com/ccxuIl3dO_z

Should I open up a new issue?

**Comment by mikejolley** [comment]
Can you paste what address the update link sends you when you click it, and from where? Just noticed multisite.

**Comment by mikejolley** [comment]
Basically I want to see if the link sends you to another site on the network which may have a separate cache.

**Comment by robertdall** [comment]
Here is the update button link for a plugin called Authorize.net AIM 
https://store.artwolfe.com/wp-admin/update.php?action=upgrade-plugin&plugin=woocommerce-gateway-authorize-net-aim%2Fwoocommerce-gateway-authorize-net-aim.php&_wpnonce=5045b950b4

Here is the update button link for the grey update button at the top of the page above all the plugins. 
https://store.artwolfe.com/wp-admin/admin.php?page=wc-addons&section=helper&filter=all&wc-helper-refresh=1&wc-helper-nonce=2be241443a

Mike I should also mention that there is no notification in the multisite network plugin view that says WooCommerce Plugins need to be updated. I am guessing same issue. 

We run on WP Engine if that helps at all. You can contact me directly if you would like and I could give you admin access to the site if required.

**Comment by mikejolley** [comment]
Can you try this and feedback on that PR? https://github.com/woocommerce/woocommerce/pull/20325
